viewer/src/dataModels/pointCloud/internal/PointCloudManager.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { PointCloudFactory } from './PointCloudFactory';
+import { PointCloudMetadataRepository } from './PointCloudMetadataRepository';
+import { PotreeGroupWrapper } from './PotreeGroupWrapper';
+import { PotreeNodeWrapper } from './PotreeNodeWrapper';
+
+export class PointCloudManager<TModelIdentifier> {
+  private readonly _pointCloudMetadataRepository: PointCloudMetadataRepository<TModelIdentifier>;
+  private readonly _pointCloudFactory: PointCloudFactory;
+
+  private _pointCloudGroupWrapper?: PotreeGroupWrapper;
+
+  constructor(
+    cadModelMetadataRepository: PointCloudMetadataRepository<TModelIdentifier>,
+    cadModelFactory: PointCloudFactory
+  ) {
+    this._pointCloudMetadataRepository = cadModelMetadataRepository;
+    this._pointCloudFactory = cadModelFactory;
+  }
+
+  resetRedraw(): void {
+    // reset is handled inside.
+  }
```

Comment:
Why have this? Would make sense if `PointCloudManager` implemented some common interface it shared with `CadManager`, but not if it's standalone as it currently is?

Reply:
I can remove it for now, but I do wish to add it at some point so the user can explicitly tell that it needs a redraw if they do something

Reply:
Easy to add when/if needed. We should avoid adding stuff for future implementation

Reply:
I removed it